packages/capabilities/src/access.js

```diff
@@ -100,3 +100,12 @@ export const session = capability({
     key: DID.match({ method: 'key' }),
   },
 })
+
+export const claim = base.derive({
+  to: capability({
+    can: 'access/claim',
+    with: DID.match({ method: 'key' }).or(DID.match({ method: 'mailto' })),
+    derives: equalWith,
+  }),
+  derives: equalWith,
```

Comment:
could use feedback ensuring these two `derives: equalWith` make sense

Reply:
The first one is obsolete, because omitting `derives` there would basically use `equalWith`. Second one is mandatory and is used to compare `claim` with `base`. Given that `claim` has no `nb` comparing just `with` fields seems like a right to me.